web/server/watcher.py

```diff
@@ -28,33 +28,41 @@ async def watch_project() -> None:
         (settings.project_path / c.METRICS).resolve(),
         (settings.project_path / c.SEEDS).resolve(),
     ]
-    ignore_entity_patterns = context.config.ignore_patterns if context else c.IGNORE_PATTERNS
-    ignore_entity_patterns.append("^\\.DS_Store$")
-    ignore_entity_patterns.append("^.*\\.db(\\.wal)?$")
-    ignore_paths = [str((settings.project_path / c.CACHE).resolve())]
-    watch_filter = DefaultFilter(
-        ignore_paths=ignore_paths, ignore_entity_patterns=ignore_entity_patterns
+    watch_filter = DefaultFilter()
```

Comment:
here i wanted to extend defaults

Reply:
I personally would have done it the following way:
```python
DefaultFilter(
    ignore_paths=DefaultFilter.ignore_paths + ignore_paths,
    ignore_entity_patterns=DefaultFilter.ignore_entity_patterns + ignore_entity_patterns,
)
```